lib/ref_arch_setup/bolt_helper.rb

```diff
@@ -2,6 +2,7 @@
 module RefArchSetup
   # Bolt helper methods
   class BoltHelper
+    FORGE_MODULE_PATH = File.dirname(__FILE__) + "/../../Boltdir/modules"
```

Comment:
If you specify a modulepath that contains modules those need to be specified otherwise they get removed. This made me think it might be better to just keep them separate, leaving the RAS modules where they are and using the Boltdir convention for the downloaded modules.